react-router.config.ts

```diff
@@ -7,4 +7,7 @@ export default {
   presets: [process.env.VERCEL ? vercelPreset() : null].filter(
     (preset): preset is Preset => Boolean(preset),
   ),
+  async prerender() {
+    return ['/terms', '/privacy'];
```

Comment:
I forgot that we have themes that we doing via cookies. I wonder how will this work with themes. Can you check which theme will be used to prerender that page and what will happen if user has selected theme other than what was used to build the page? We should check what happens for both direct navigation to the page by typing terms or privacy path directly into url and also via client side navigation